pages/agent/v3/aws/autoscaling_mac_metal.md.erb

```diff
@@ -0,0 +1,200 @@
+[Amazon EC2 Dedicated Hosts Pricing](https://aws.amazon.com/ec2/dedicated-hosts/pricing/)
+for details on which regions have `mac1.metal` dedicated hosts.
+
+See also the [Amazon EC2 Mac instances User Guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-mac-instances.html)
+for more details on AWS EC2 Mac instances.
+
+## Choose a VPC Layout
+
+Before deploying this template you must choose a VPC subnet design, and which
+VPC security groups your instances should belong to.
+
+Depending on your threat model, you may find running instances in your default
+VPC’s public subnets with a public IP address suitable. Otherwise, you may wish
+to explore options like separate Public/Private subnets and a NAT Gateway, and
+using a Bastion or VPN to access the instances over SSH and VNC.
+
+In supported regions, `mac1.metal` dedicated hosts may not be available in
+every Availability Zone. Consider using a subnet in every Availability Zone to
+maximise the pool of instances available to boot from.
+
+You also need to configure or define the VPC Security Groups your instance
+network interfaces should belong to. At a minimum, inbound SSH access is
+required to set up your initial template AMI.
+
+## Build an AMI
+
+Before deploying this template, you must create a template AMI that can be
+horizontally scaled across multiple instances.
+
+1. Reserve a [AWS mac1.metal](https://aws.amazon.com/ec2/instance-types/mac/)
+Dedicated Host.
+1. Boot a macOS instance using your desired AMI on the Dedicated Host.
+1. Configure instance VPC subnets, security groups, and key pairs so that you
+can access the instance.
+1. Using an SSH or SSM session:
+	* Set a password for the ec2-user using `sudo passwd ec2-user`
+	* Enable screen sharing using `sudo /System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart -activate -configure -access -on -restart -agent -privs -all`
+1. Using a VNC session:
+	1. Sign in as the `ec2-user`
+	1. Enable *Automatic login* for the ec2-user in *System Preferences* > *Users & Accounts* > *Login Options*
+	1. Disable *Require password* in *System Preferences* > *Security & Privacy* > *General*
+1. Install your required version of Xcode, and ensure you launch Xcode at least
+once so you are presented with the EULA prompt.
+1. Using the AWS EC2 Console, create an AMI from your instance.
+
+You do not need to install the `buildkite-agent`, it is automatically installed
+by the Launch Template `UserData` script.
+
+## Associate your AMI with a Customer managed license in AWS License Manager
+
+To launch an instances using a Host Resource Group, the instance AMI must be
+associated with a Customer managed license in *AWS License Manager*.
+
+Using the AWS Console, open the *AWS License Manager* and navigate to
+*Customer managed licenses*. Create a new Customer managed license, enter a
+descriptive name and select a License Type of `Cores`.
```

Comment:
If License Type is an actual GUI label (e.g. field name), italicise. Otherwise, lowercase.